sky/services/platform/path_provider.mojom

```diff
@@ -0,0 +1,38 @@
+// Copyright 2016 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+[DartPackage="sky_services"]
+module flutter.platform;
+
+/// Returns commonly used locations on the filesystem.
+interface PathProvider {
+  /// Path to the read-only directory that contains the assets packaged with
+  /// the application bundle.
+  ///
+  /// Example(s):
+  ///    iOS: The application bundle directory.
+  AssetsDirectory() => (string path);
```

Comment:
I'm not entirely sure we want to expose this as a path. For some abstractions, this content isn't available in a literal directory. Is there client code that can't use AssetBundle?